FatturaBase.cs

```diff
@@ -23,8 +23,52 @@ public override void WriteXml(XmlWriter w)
             base.WriteXml(w);
             w.WriteEndElement();
         }
+        //public static FatturaBase CreateInstanceFromXml(string xmlPath)
```

Comment:
toglierei questo codice commentato. Come immagino sia tua intenzione, una volta fatta la merge di questa PR si può aggiungere questa feature a FE.Extensions.

Reply:
si certo, considera il mio codice più come uno spunto. Il metodo richiede una dipendenza in più infatti, se serve lo mettiamo nelle Extensions